client/src/components/SearchLocation.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled from "@emotion/styled";
+import StyledSpan from "./StyledSpan";
+import InputField from "./InputField";
+import InputSlider from "./InputSlider";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+`;
+
+const Label = styled.label`
+  font-size: 1.3rem;
+  display: flex;
+  margin-top: 30px;
+  align-items: center;
+`;
+const Value = styled.span`
+  justify-content: flex-end;
+  display: flex;
+`;
```

Comment:
Maybe you could make small components for the Wrapper, the Label and the Value (like you made it with the StyledSpan).
You use it in `SearchExperience`, `SearchProfession` and here.